guide/resources/wallet-interoperability.md

```diff
@@ -0,0 +1,196 @@
+   width = 400
+   height = 300
+   layout = "float-right-desktop"
+%}
+
+Transaction data is stored on the Bitcoin block chain and available in any wallet a user has set up. As is, this transaction data does not contain any useful information about the reasons why transactions were made, who owns each address, etc. To better understand and organize their finances, users typically enrich transaction data by assigning contacts, notes, labels, and other useful information. This data should be stored in standardized, open formats and easily synced between applications. This is especially useful for users who rely on multiple devices.
+
+</div>
+
+## Transaction files
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/wallet-interoperability/transaction-file.jpg"
+   retina = "/assets/images/guide/wallet-interoperability/transaction-file@2x.jpg"
+   alt-text = "Transfer of transaction files between applications"
+   width = 400
+   height = 300
+   layout = "float-right-desktop"
+%}
+
+Also known as the [Partially Signed Bitcoin Transaction Format](https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki), this type of file allows for storage and transfer of transactions that are not finalized. Common use cases are in multi signature wallets where one application prepares a transaction and sends it to another wallet for an additional signature. It also allows signig wallets to be offline, as the transaction data can be transferred via USB drives or animated QR codes.
+
+</div>
+
+## Multi signature wallets
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/wallet-interoperability/multi-signature.jpg"
+   retina = "/assets/images/guide/wallet-interoperability/multi-signature@2x.jpg"
+   alt-text = "Cosigner wallets interacting with a multi signature wallet"
+   width = 400
+   height = 300
+   layout = "float-right-desktop"
+%}
+
+Per definition, multi signature wallets require interaction between all cosigners and therefore interoperability between their software (or hardware) of choice. Especially when there are multiple cosigners (rather than one owner using multiple signing devices), there should be strong design considerations to how the cosigners exchange keystores and transaction data. 
+
+</div>
+
+## Self-hosted nodes
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/wallet-interoperability/node-options.jpg"
+   retina = "/assets/images/guide/wallet-interoperability/node-options@2x.jpg"
+   alt-text = "Node options"
+   width = 400
+   height = 300
+   layout = "float-right-desktop"
+%}
+
+While it is extremely convenient when applications provide their own node connections, it is recommended to allow users to have a choice, and potentially even encourage them to set up their own node. This results in better decentralization, and also has privacy and security benefits.
```

Comment:
Important point :clap:

Reply:
Would be a good spot to link to a "Principles" or "Values" section.